include/zephyr/bluetooth/classic/classic.h

```diff
@@ -38,10 +38,20 @@ extern "C" {
  * @{
  */
 
+/** @brief BR/EDR discovery private structure */
+struct bt_br_discovery_priv {
+	/** Clock offset */
+	uint16_t clock_offset;
+	/** Page scan repetition mode */
+	uint8_t pscan_rep_mode;
+	/** Resolving remote name*/
+	uint8_t resolving;
+} __packed;
```

Comment:
I don't think you need the `__packed`. It's also confusing, since this is normally reserved for protocol structs.